### ISSUE TITLE
Show additional info (crate and or duration) in test titles

### ISSUE THREAD

**Issue by Canop** [Canop/bacon#218]
Rationale for showing the crate given [here](https://github.com/Canop/bacon/pull/212#issuecomment-2385297153) by @DragonDev1906.

The duration would be only available when it can be parsed from the output (i.e. when the tests are done with nextest). 

The reason I'm a little reluctant is I try to keep the display uncluterred for easier reading, and I try to make bacon usable in a thin terminal.

Opinions welcome.

**Comment by DragonDev1906** [comment]
> and I try to make bacon usable in a thin terminal.

Good point with the thin terminal. How difficult would it be to only show this extra information (crate + time) if the terminal is wide enough? That way you could temporarily full-screen the terminal and get more information.

**Comment by Canop** [comment]
I made it a separate issue because I'll try to merge the PR #212 so that I can more easily manage various small issues.

**Comment by DragonDev1906** [comment]
> But this kind of collides wit the wrapping feature: what if people start to want the crate, duration even when it means the line is wrapped?

I doubt that's the case, as at that point the rest of the issue description gets difficult enough to read that you probably don't want even more information. Unless you have really long module names/import paths, so perhaps having it configurable does make sense.

![2024-10-01-180014_496x321_scrot](https://github.com/user-attachments/assets/223ae1c8-c546-494f-9d64-cbb37dd39405)
